plugins/cue_points/quiz/lib/enums/QuizCapabilityName.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * @package plugins.quiz
+ * @subpackage lib.enum
+ */
+class QuizCapabilityName implements IKalturaPluginEnum, CapabilityName
+{
+
```

Comment:
Dynamic enums should define their own constants, don't use the plugin name instead.